training/computational-chemistry/gromacs/gromacs-test.yml

```diff
@@ -9,4 +9,7 @@
     gro_output:
       asserts:
         has_text:
-          text: "LYSOZYME in water"
\ No newline at end of file
+          text: "LYSOZYME in water"
+    xtc_output:
+      file: traj.xtc
```

Comment:
Can we download this fle from somewhere? Zenodo, cargo-port etc ...

Reply:
It seems a bit crazy to upload a 13 MB file to Zenodo just to do a `sim_size` check. Would be nice if there was a Galaxy test option like `check_size=13000000` but I know this has been discussed before.